radixdlt-core/radixdlt/src/main/java/com/radixdlt/client/Address.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * (C) Copyright 2021 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ */
+
+package com.radixdlt.client;
+
+import com.radixdlt.crypto.ECPublicKey;
+import com.radixdlt.crypto.exception.PublicKeyException;
+import com.radixdlt.utils.Bits;
+import org.bitcoinj.core.Bech32;
+
+public class Address {
+	private Address() {
+		throw new IllegalStateException();
+	}
+
+	public static String ofValidator(ECPublicKey key) {
+		var bytes = key.getCompressedBytes();
+		var convert = Bits.convertBits(bytes, 0, bytes.length, 8, 5, true);
+		return Bech32.encode("vb", convert);
+	}
+
+	public static ECPublicKey parseValidatorAddress(String v) {
+		var bech32Data = Bech32.decode(v);
+		if (!bech32Data.hrp.equals("vb")) {
+			throw new IllegalArgumentException();
+		}
+		var keyBytes = Bits.convertBits(bech32Data.data, 0, bech32Data.data.length, 5, 8, false);
```

Comment:
I would not use `convertBits` inline like this, (5, 8, false) and (8, 5, true) are the tuple inputs we wanna use always when using Bech32, so I would rework this:

```java
	public static String ofValidator(ECPublicKey key) {
		var bytes = key.getCompressedBytes();
		var bech32Data = toBech32Data(bytes);
		return Bech32.encode("vb", bech32Data);
	}
	
	public static byte[] toBech32Data(byte[] bytes) {
		return Bits.convertBits(bytes, 0, bytes.length, 8, 5, true);
	}

	public static byte[] fromBech32Data(byte[] bytes) {
		return Bits.convertBits(bytes, 0, bytes.length, 5, 8, false);
	}

	public static ECPublicKey parseValidatorAddress(String v) {
		var bech32Data = Bech32.decode(v);
		if (!bech32Data.hrp.equals("vb")) {
			throw new IllegalArgumentException();
		}
		var keyBytes = fromBech32Data(bech32Data.data);
		try {
			return ECPublicKey.fromBytes(keyBytes);
		} catch (PublicKeyException e) {
			throw new IllegalStateException();
		}
	}
```

Reply:
and maybe move `toBech32Data` and `fromBech32Data` to some better location...